networkvisualizer/network_visualizer.go

```diff
@@ -153,11 +156,11 @@ func AssembleGraph(start int, end int, chains []Chain) {
 func AddCoincident(chains []Chain, hash common.Hash) {
 	uncle := true
 	for i := 0; i < len(chains); i++ {
-		_, err := chains[i].client.HeaderByHash(context.Background(), hash)
+		header, err := chains[i].client.HeaderByHash(context.Background(), hash)
 		if err == nil {
-			chains[i].AddNode(hash, 0)
+			chains[i].AddNode(hash, 0, header)
 			if chains[i].order < 2 {
-				chains[i].AddEdge(false, fmt.Sprintf("%d", chains[i].order)+hash.String()[2:7], fmt.Sprintf("%d", chains[i].order+1)+hash.String()[2:7])
+				chains[i].AddEdge(false, fmt.Sprintf("%d", chains[i].order)+hash.String()[2:10], fmt.Sprintf("%d", chains[i].order+1)+hash.String()[2:10], "")
```

Comment:
(nit) in general, its preferred to avoid magic numbers like this, _especially_ when they seem to be repeated. You might consider defining some constants at the top of the file that let you tweak details like this (i.e. hash prefix length, etc).

That would also make things easier to parameterize, if we wanted to have CLI options to change those values... but that is not something to take on in this PR, haha